### ISSUE TITLE
[CAIA A11y Research - PILOT]: VA BTSSS Travel Pay

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#81037]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATE:**  `4/18/2024`
> - **TYPES OF ASSISTIVE TECH:** `Mac+voiceover`

# CAIA A11y Support
This is a child ticket of the research ticket [80740](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80740) - created to track work, completed from this pilot session.

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

@coforma-jamie 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: VA BTSSS (Travel Pay)
- OCTO-DE product owner: Kay Lawyer
- Product manager: Ayush Chakravarty
- Designer: Micah Taylor
- FE engineer: Kevin Duensing
- Product/team Slack channel:#beneficiary-travel-team

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** `4/18/2024`
- **Prototype**: [Prototype in CodePen](https://codepen.io/team/VA-gov-Applications/project/live/DQmqxR)
- **Research Convo Guide**: [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/beneficiary-travel/research/2024-04-Reimbursement-Status-Look-and-Feel-Study/ConversationGuide.md)
- **Research Plan**: [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/beneficiary-travel/research/2024-04-Reimbursement-Status-Look-and-Feel-Study/ResearchPlan.md)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** @coforma-jamie
- [x] **Determine date and time for pilot:** Noon 4/18/2024
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [ ] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/beneficiary-travel/research/2024-04-Reimbursement-Status-Look-and-Feel-Study/ConversationGuide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] **Review prototype** in `CodePen` - `Available`
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. Folder for Team in GH Deliverables for CAIA A11Ys.
```
